resource/stopwords/ko.txt

```diff
@@ -0,0 +1,55 @@
+가
```

Comment:
Korean isn't supported, cf. https://github.com/dataiku/dss-plugin-nlp-visualization/blob/9572eb5e0540614f52cb739ae114022c7ffcbd6e/python-lib/language_dict.py#L67

Reply:
Yep, good catch. It's less likely that we'll be able to solve the installation issues inherent to the Korean tokenization library (external open-source lib). I would rather keep the stopwords here, as it means we could provide Korean support to customers who request it, with a simple fork and some consulting help to install the Korean dependencies on the server